prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctionsTests.kt

```diff
@@ -678,19 +718,19 @@ class CustomFHIRFunctionsTests {
         assertThat(age is Age).isEqualTo(true)
         if (age is Age) {
             assertThat(age.unit).isEqualTo("day")
-            assertThat(age.value.toInt()).isEqualTo(0)
+            assertThat(age.value.toInt()).isEqualTo(4)
             assertThat(age.code).isEqualTo("d")
         }
     }
 
     @Test
-    fun `test convertDateToAge 0 days - default`() {
+    fun `test convertDateToAge 4 days - default`() {
```

Comment:
I think we can similarly combine some of these other tests with the same kind of `forEach` block as my suggestion above. Consider it optional!